### PR TITLE
[15.0][MIG] partner_data_vies_populator

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: partner_data_vies_populator/tests/test_partner_data_vies_populator.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2015 Forest and Biomass Romania
+# Copyright (C) 2020 NextERP Romania
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestPartnerCreateByVAT(TransactionCase):
+    def setUp(self):
+        super().setUp()
```
Comment (rousseldenis) [inline partner_data_vies_populator/tests/test_partner_data_vies_populator.py]: Could you switch to class method ?

File: partner_data_vies_populator/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+* `NextERP Romania <https://www.nexterp.ro>`_:
+
+  * Fekete Mihai <feketemihai@nexterp.ro>
+
+* `Forest and Biomass Romania <https://www.forbiom.eu/>`_
+
+* `Akretion <https://akretion.com/>`_ :
+
+  * Alexis de Lattre <alexis.delattre@akretion.com>
```
Comment (rousseldenis) [inline partner_data_vies_populator/readme/CONTRIBUTORS.rst:9]: @jdidderen-noviat You can add yourself here

**Review by rousseldenis — APPROVED** [review]
LGTM. Minor comment

File: partner_data_vies_populator/tests/test_partner_data_vies_populator.py (hunk 1 above)
Comment (ghost) [reply partner_data_vies_populator/tests/test_partner_data_vies_populator.py]: @rousseldenis And it's done. Thanks for your review :)

**Review by dreispt — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please don't add merge commits that mess up the branch history

**Comment by dreispt** [comment]
@pedrobaeza I interrupted it. Why will the additional commit mess the history?

**Comment by pedrobaeza** [comment]
The problem is the merge commits. You must rebase instead.

**Comment by dreispt** [comment]
@pedrobaeza That was trying a manual conflict resolution.
I'm surprised Github make a merge commit out of that.
Anyway, there are precommit problems and I'll have to create a superseding PR 😞

**Comment by dreispt** [comment]
Superseded by #1641

**Pull request closed without merging**